packages/core-app-api/src/routing/RouteTracker.tsx

```diff
@@ -68,8 +70,9 @@ const getExtensionContext = (
 
     return {
       extension: 'App',
-      pluginId: routeObject.plugin?.getId() || 'root',
+      pluginId: plugin?.getId() || 'root',
       ...(routeRef ? { routeRef: (routeRef as { id?: string }).id } : {}),
+      _element: routeObject.element as string,
```

Comment:
I think it's best to move over to some other name than `element` here, we just use that name because it's already part of the routing object in react-router. Maybe something like `_routeNodeType`?